api/receiver/receive_results.go

```diff
@@ -63,13 +63,17 @@ func HandleResultsUpload(a AppEngineAPI, w http.ResponseWriter, r *http.Request)
 	// Non-existent keys will have empty values, which will later be
 	// filtered out by scheduleResultsTask.
 	extraParams := map[string]string{
-		"labels": r.PostFormValue("labels"),
+		"labels": r.FormValue("labels"),
 		// The following fields will be deprecated when all runners embed metadata in the report.
-		"revision":        r.PostFormValue("revision"),
-		"browser_name":    r.PostFormValue("browser_name"),
-		"browser_version": r.PostFormValue("browser_version"),
-		"os_name":         r.PostFormValue("os_name"),
-		"os_version":      r.PostFormValue("os_version"),
+		"revision":        r.FormValue("revision"),
+		"browser_name":    r.FormValue("browser_name"),
+		"browser_version": r.FormValue("browser_version"),
+		"os_name":         r.FormValue("os_name"),
+		"os_version":      r.FormValue("os_version"),
+	}
+	callbackURL := r.FormValue("callback_url")
+	if callbackURL != "" {
```

Comment:
No need to have this check. This is already done in https://github.com/web-platform-tests/wpt.fyi/blob/3b1fb7aaa4517c1ea7e02e486c760ee2fc4b40e7/api/receiver/appengine.go#L131

Reply:
Removed check.